crates/lsp/src/tower_lsp.rs

```diff
@@ -188,27 +179,20 @@ where
     I: AsyncRead + Unpin,
     O: AsyncWrite,
 {
-    log::trace!("Starting LSP");
-
     let (service, socket) = new_lsp();
     let server = tower_lsp::Server::new(read, write, socket);
     server.serve(service).await;
-
-    log::trace!("LSP exiting gracefully.",);
```

Comment:
Again these messages are "too soon" or "too late" from the perspective of the log thread

Reply:
Could be `eprintln!()` then

Reply:
I feel like it would be quite odd to have two messages get sent to stderr when all other messages end up over `window/logMessage`. I feel like it just adds one more layer of places to look for logs. Would it be ok to try and drop these for now and see if we need them later on?

Reply:
Sure that's fine with me.